src/GraphQL.Conventions/Types/Resolution/ObjectReflector.cs

```diff
@@ -356,7 +356,17 @@ private static bool HasValidReturnType(MemberInfo memberInfo)
             {
                 return true;
             }
-            return returnType != typeof(object);
+                   
+            return 
+                returnType != typeof(object) && 
+                ObjectReflectorSettings.IgnoreFieldsWithVoidReturnType ? returnType != typeof(void) : true;
         }
     }
+
+    internal static class ObjectReflectorSettings
+    {
+        public static List<string> IgnoredNamespaces { get; } = new List<string>() { nameof(System) + "." };
+
+        public static bool IgnoreFieldsWithVoidReturnType { get; set; } = false;
```

Comment:
See above; let us _always_ ignore fields returning `void`.

Reply:
Noted, please.  Made it ```false``` to avoid breaking changes.  It did broke some of your tests, actually.

Reply:
https://github.com/graphql-dotnet/conventions/pull/78 ignores all members with a `void` return type, and all unit tests are passing :-)